docs/developers/contributing/simple-module.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: standardheadmd
+title: '- Nucleus Developers'
+header: Creating a Simple module
+---
+
+## Explanation
+A simple module is a module which does not have any configuration whatsoever. You can add commands, listeners, and tasks to a module. In 
+order to begin creating a module you must first locate the src/main/java/io/github/nucleuspowered/nucleus/modules package. Next a new
+package must be created which will contain the new module. To finish with creating a very basic module all that needs doing is creating a 
```

Comment:
Also, might be worth mentioning that the `plugin` variable is automatically available for use.